test/e2e/e2e.go

```diff
@@ -585,9 +585,13 @@ func getApiAddress() string {
 func IsGatewayModeLocal() bool {
 	anno, err := e2ekubectl.RunKubectl("default", "get", "node", "ovn-control-plane", "-o", "template", "--template={{.metadata.annotations}}")
 	if err != nil {
+		framework.Logf("Error getting annotations: %v", err)
 		return false
 	}
-	return strings.Contains(anno, "local")
+	framework.Logf("Annotations received: %s", anno)
+	isLocal := strings.Contains(anno, "local")
```

Comment:
btw, what happens if theres a network named local in the annotations ? fine for now since its how it was before you came into it.

Reply:
I can change this to using ENV variable for GATEWAY_MODE there is another util and another place which use that..but I thought you wanted to move away from that we can also be very explicit and match for mode:local